kubeflow/core/jupyterhub_spawner.py

```diff
@@ -91,3 +91,18 @@ def extra_resource_limits(self):
 # How much disk space do we want?
```

Comment:
Do we need to uncomment the line above that sets singleuser_fs_gid
https://github.com/kubeflow/kubeflow/pull/22#issuecomment-350500944

Did you test if this works?

Reply:
yup, just tested more thoroughly:
1. created notebook in work dir
2. deleted jupyter pod
3. recreated pod
4. notebook was there

Reply:
AD. singleuser_fs_gid - just checked - we run container as root. That makes it work without gid gimmicks but it's different issue on it's own:)